arviz/tests/helpers.py

```diff
@@ -11,8 +11,12 @@
 import pyro.distributions as dist
 from pyro.infer.mcmc import MCMC, NUTS
 import pystan
+import tensorflow_probability as tfp
```

Comment:
If you have some time could you try changing the import order to see if that fixes things, per your research and @ColCarroll's? 

If you don't that have time thats ok! We can do what @ColCarroll suggested which is merge and fix later

Reply:
In fact I tried and got a segment fault error.

Reply:
That's frustrating, thank you for trying!